src/asciidoctorWebViewConverter.ts

```diff
@@ -183,6 +187,17 @@ export class AsciidoctorWebViewConverter {
       const title = node.hasAttribute('title') ? ` title="${node.title}"` : ''
       return `<a href="${href}"${id}${role}${title} data-href="${href}">${node.text}</a>`
     }
+    // if Antora support activated
+    if (nodeName === 'image') {
+      const nodeAttributes = node.getAttributes()
+      const target = nodeAttributes.target
+      const resolvedPath = resolveAntoraImageIds(target, this.contentCatalog, this.src)
+      if (resolvedPath !== undefined) {
+        const alt = node.hasAttribute('alt') ? ` alt="${nodeAttributes.alt.split(':').pop()}"` : ''
+        // TODO check for Antora stylesheet
+        return `<img src= "${resolvedPath}" ${alt}>`
```

Comment:
do not implement this part for now, fallback to the `baseConverter`, we will implement this logic later